### PR TITLE
Initial pass at dashboard integration tests

### PR DESCRIPTION
**Pull request by akeem**
Starting to add very basic coverage around the dashboard somewhat loading correctly.

* Includes associated update to holes found in edge cases while testing.

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: unlock-app/src/middlewares/storageMiddleware.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (cellog) [inline unlock-app/src/middlewares/storageMiddleware.js]: I think 2 unintended files made it into this commit?

Comment (akeem) [reply unlock-app/src/middlewares/storageMiddleware.js]: This is me updating spots where the code is now bleeding, the tldr is that the code was updated  in a way that starts to breaks subtly if Locksmith isn't available. A 404 isn't a fatal issue, but not handling a promise rejection might be

Comment (cellog) [reply unlock-app/src/middlewares/storageMiddleware.js]: gotcha. 👍
